### ISSUE TITLE
`dollar-variable-pattern` doesn't work inside mixins

### ISSUE THREAD

**Issue by nickbullock** [stylelint-scss/stylelint-scss#953]
Example reproduction:

stylelintrc with kebab case regexp:
```
"scss/dollar-variable-pattern": "^[a-z]+(?:-[a-z]+)*$",
```

```
@myFontSize($fontSize) {
 font-size: $fontSize;
}
```

I believe it should report error.

**Comment by kristerkari** [comment]
Thanks!

That's correct, the rule does not support function arguments.

I guess it would make sense to add support for fn arguments, since even though they're technically function arguments and not variables, the syntax is identical.
